Cargo.toml

```diff
@@ -14,4 +14,4 @@ members = [
 [patch.crates-io]
 # patch for build script bug preventing bazel build
 # see https://github.com/rust-lang/rustc_apfloat/pull/17
-rustc_apfloat = { git = "https://github.com/redsun82/rustc_apfloat.git", rev = "096d585100636bc2e9f09d7eefec38c5b334d47b" }
+rustc_apfloat = { git = "https://github.com/redsun82/rustc_apfloat.git", rev = "32968f16ef1b082243f9bf43a3fbd65c381b3e27" }
```

Comment:
It looks like your PR was merged 4 days ago. https://github.com/rust-lang/rustc_apfloat/pull/17 I suppose we still need this patch for a little while, but hopefully we can drop it once a new version of rustc_apfloat is published

Reply:
yes, I don't know how long it will be till my patch gets released